Indicators/CompositeIndicator.cs

```diff
@@ -171,6 +177,12 @@ private void ConfigureEventHandlers()
                     newLeftData = null;
                     newRightData = null;
                 }
+                else
+                {
+                    // Fill forward time
+                    Current = (IndicatorDataPoint)Current.Clone();
+                    Current.Time = MaxTime(updated);
```

Comment:
Cool! We need a nice collection of tests around this. 
I think during _each_ update, we will always come through this `if` because Right/Left not updated sequentially but first one and later the other? Maybe we need some extra logic for that case?

Reply:
I just added two tests to show how the time on the consolidator advances. One of them `TimeAdvancedWithDataToOnlyOneSide`  is red/green'd from master to this branch. I am also adding one now for checking the data point after one update (since it shouldn't change until both are received) and after.

The pre existing `if` statement that I added the `else` to takes care of them not being updated sequentially by tracking new datapoints for either side, once both have a new datapoint it will updated.